packages/test/test-end-to-end-tests/src/test/gc/gcStats.spec.ts

```diff
@@ -97,6 +97,12 @@ describeNoCompat("Garbage Collection Stats", (getTestObjectProvider) => {
 			attachmentBlobCount: 2,
 			unrefAttachmentBlobCount: 0,
 			updatedAttachmentBlobCount: 2,
+			lifetimeNodeCount: 9,
+			lifetimeDataStoreCount: 3,
+			lifetimeAttachmentBlobCount: 2,
+			deletedNodeCount: 0,
```

Comment:
I notice there are no tests in here for Sweep.  What are the pros/cons of that in your mind?  Should we open a small follow-up task?  If we feel it's worth having this e2e test (not just unit tests) we should also include Sweep.  Alternative would be to delete this file if it's redundant with the unit tests, just more complex.

Reply:
AB#6403